lib/core/src/Cardano/Wallet/Primitive/Types/UTxOIndex/Internal.hs

```diff
@@ -197,15 +197,15 @@ singleton i o = insertUnsafe i o empty
 -- and all others will be ignored.
 --
 fromSequence :: Foldable f => f (TxIn, TxOut) -> UTxOIndex
-fromSequence = flip insertMany empty
+fromSequence = flip insertMany mempty
```

Comment:
Using `mempty` here is okay, because the context makes it obvious what the type is.

Just to emphasise that I'm not against `mempty` per-se! 😄